.github/workflows/ci-check-repo.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check Formatting and Commiters
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  verify:
+    name: Verify format and commiters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go 1.13
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+        id: go
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check all
+        working-directory: ./go
+        # Keep this in sync with //go/utils/prepr/prepr.sh.
```

Comment:
Can we update `prepr.sh` to say "Keep this in sync with `//.github/workflows/ci-check-repo.yaml`?